adcc/ExcitedStates.py

```diff
@@ -348,8 +348,8 @@ def plot_spectrum(self, broadening="lorentzian", xaxis="eV",
             ["eV", "au", "nm", "cm-1"]
         yaxis : str
             Quantity to plot on the y-Axis. Options are "cross_section",
-            "osc_strength", "dipole" (plots norm of transition dipole),
-            "rotational_strength" (ECD spectrum with rotational strength)
+            "oscillator_strength", "dipole" (plots norm of transition dipole),
+            "rotatory_strengths" (ECD spectrum with rotational strength),
```

Comment:
`_strength` or `_strengths`?
Or both should be accepted...

Reply:
Probably plural, then it's exactly as the properties. Yes we should probably accept both.

Reply:
Now it should be the singular form 😄